cli_reference/helm_cli/configuring-custom-helm-chart-repositories.adoc

```diff
@@ -0,0 +1,14 @@
+[id="configuring-custom-helm-chart-repositories"]
+
+= Configuring custom Helm chart repositories
+include::modules/common-attributes.adoc[]
+:context: configuring-custom-helm-chart-repositories
+
+toc::[]
+
+The *Developer Catalog*, in the *Developer* perspective of the web console, displays the Helm charts available in the cluster. By default, it lists the Helm charts from the link:https://redhat-developer.github.io/redhat-helm-charts/index.yaml[Red Hat Helm chart repository]. As a cluster administrator, you can add multiple Helm chart repositories, apart from the default one, and display the Helm charts from these repositories in the *Developer Catalog*.
```

Comment:
Is the link to the Red Hat Helm chart repository intended to open a file or direct to the actual repo?

Reply:
Modified this a bit.